src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -4,52 +4,123 @@
 using System.Linq;
 using System.Net;
 using System.Threading;
+using System.Data.SQLite;
 using NLog;
 using NLog.Common;
 using NLog.Targets;
 using NzbDrone.Common.EnvironmentInfo;
-using SharpRaven;
-using SharpRaven.Data;
+using Sentry;
+using Sentry.Protocol;
 
 namespace NzbDrone.Common.Instrumentation.Sentry
 {
     [Target("Sentry")]
     public class SentryTarget : TargetWithLayout
     {
-        private readonly RavenClient _client;
+        // don't report uninformative SQLite exceptions
+        // busy/locked are benign https://forums.sonarr.tv/t/owin-sqlite-error-5-database-is-locked/5423/11
+        // The others will be user configuration problems and silt up Sentry
+        private static readonly HashSet<SQLiteErrorCode> FilteredSQLiteErrors = new HashSet<SQLiteErrorCode> {
+            SQLiteErrorCode.Busy,
+            SQLiteErrorCode.Locked,
+            SQLiteErrorCode.Perm,
+            SQLiteErrorCode.ReadOnly,
+            SQLiteErrorCode.IoErr,
+            SQLiteErrorCode.Corrupt,
+            SQLiteErrorCode.Full,
+            SQLiteErrorCode.CantOpen,
+            SQLiteErrorCode.Auth
+        };
 
-        private static readonly IDictionary<LogLevel, ErrorLevel> LoggingLevelMap = new Dictionary<LogLevel, ErrorLevel>
-        {
-            {LogLevel.Debug, ErrorLevel.Debug},
-            {LogLevel.Error, ErrorLevel.Error},
-            {LogLevel.Fatal, ErrorLevel.Fatal},
-            {LogLevel.Info, ErrorLevel.Info},
-            {LogLevel.Trace, ErrorLevel.Debug},
-            {LogLevel.Warn, ErrorLevel.Warning},
+        // use string and not Type so we don't need a reference to the project
+        // where these are defined
+        private static readonly HashSet<string> FilteredExceptionTypeNames = new HashSet<string> {
```

Comment:
@markus101 Do we really want to block any log reporting for these?
Iirc we already rate-limited boot loops a bit.

Things like TinyIoCResolutionException could be a deployment issue we'd want to know about.

It might be better if we could make something survive the reboot and block the same message from being reported again. file in appdata, tmp, anything really.

Reply:
Good point, `TinyIoCResolutionException` is a great example of something that could have gone wrong in a deploy (missed binary or something missing on the platform), it's unlikely it'd get deployed with the testing in place, but we don't cover every base.

We might be able to do rate limiting on the sentry side, I know you did some work to avoid constant restarts if the app fails to load, would that cover us and anything else we rate limit? Not opposed to your suggested approach to avoid reporting the same thing over and over.